IAteIt/IAteIt/View/Feed/Component/FeedFooterView.swift

```diff
@@ -20,7 +20,12 @@ struct FeedFooterView: View {
                         .fontWeight(.semibold)
                 }
                 if meal.comments?.count ?? 0 > 0 {
-                    Text("View all comments")
+                    Text("View all comments \(meal.comments?.count ?? 0)")
```

Comment:
```suggestion
                    Text("View all \(meal.comments?.count ?? 0) comments ")
```
문구가 "View all 2 comments"와 같이 되는 것이 문법에 맞을 것 같습니다!

Reply:
동의합니닷